src/_core/usdPrices.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  createContext,
+  ReactNode,
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import invariant from 'tiny-invariant';
+
+import { usePopupSelector } from '../popup/store/react';
+import Background from '../ui/services/Background';
+import { startPolling } from './polling';
+import { useDebouncedValue } from './useDebouncedValue';
+
+const UsdPricesContext = createContext<
+  null | ((assetIds: string[]) => undefined | (() => void))
```

Comment:
Values like `null` & `undefined` seems to be much more expected at the end of a union, not at the beginning)